src/utilities/focus.ts

```diff
@@ -133,10 +133,11 @@ export function getNextElement(
 export function isElementVisible(element: HTMLElement): boolean {
   return (
```

Comment:
it might be more readable to do this:

```typescript
// If the element is not valid, return false.
if (!element || !element.getAttribute) {
return false;
}

let visibilityAttribute = element.getAttribute(IS_VISIBLE_ATTRIBUTE);

// If the element is explicitly marked as not visible, return false.
if (visibilityAttribute !== null)) {
  return Boolean(visibilityAttribute);
}

// Fallback to other methods of determining actual visibility.
return (
...
);
```